misc/glossary.md

```diff
@@ -0,0 +1,156 @@
+
+**Growth Scope:** Primary Scope that interacts with the growth increasing the demand of Dai and all of the distribution channels in the Maker protocol that deploy human and financial capital to achieve said demand.
+
+**Home Base:** L2 shard with an active Maker Vault Engine that Creator DAOs choose.
+
+**Infrastructure Scope:** Supporting Scope that interacts with the most critical security aspects of the Maker Protocol. This includes smart contracts, the keepers, the oracles, and other node network infrastructure.
+
+**Interface Scope:** Supporting Scope that regulates and develops the core elements of frontend concepts.
+
+**Maker Core:** Core protocol with minimal governance upon which the Maker Ecosystem is built. 
+
+**Maker Ecosystem:** Ecosystem consisting of SubDAOs, governance, ecosystem actors, and users.
+
+**Maker Teleport:** Module that allows users to instantly transfer Dai from the Ethereum Mainnet to L2s or from one L2 to another L2.
+
+**Meta:** Concept describing the second-order and contextual effects surrounding a particular subject.
+
+**Miscellaneous RWA:** Centralized on-chain RWA assets such as WBTC, cross-chain bridge tokens from L1s, and security token vaults.
+
+**Oracle Services:** Services related to the oracle interface that communicates real world data to smart contracts.
+
+**Phoenix Stance:** Endgame Stance where Dai does not allow RWA except for Physically Resilient RWA.
+
+**Physical Resilience:** Supporting Scope that is involved with RWA that cannot be easily seized.
+
+**Physically Resilient RWA:** RWA Asset that cannot be easily seized via legal means.
+
+**Pigeon Stance:** Endgame Stance that has no limit on RWA and where Dai remains pegged to 1 USD.
+
+**Primary Scopes:** Scope Category that includes Stability and Liquidity, Decentralized Collateral, RWA Collateral, and Ecosystem Operations.
+
+**Protector DAO:** Type of SubDAO responsible for real-world assets.
+
+**Protocol Engineering Scope:** Supporting Scope that is responsible for the functionality and upgrades of the core protocol.
+
+**Protocol-Owned Vault:** Maker vault that can only be utilized by Maker Governance that uses decentralized collateral to generate Dai.
+
+**RWA Adopted Vault:** Vault with RWA collateral that only pays a fixed part of their stability fee to Maker Core. A Protector SubDAO is responsible for liquidations and other losses before Maker Core.
+
+**RWA Collateral Scope:** Primary Scope that interacts with real world assets that include: cashlike (centralized stable coins & bonds), clean money (renewable energy projects), physical resilient, and miscellaneous (WBTC) assets.
+
+**RWA Services:** External services that require interaction with the physical world in order to obtain assets.
+
+**Scope:** Broad focus area of work within which SubDAOs, Ecosystem Actors and Maker Governance operate.
+
+**Scope Framework:** Set of rules that SubDAOs within that Scope must adhere to. 
```

Comment:
I think they now cover more than this in practice. Also constraining Maker Core? Maybe more stuff?